submissions/ebojudahCherish/html-forms/contact-form.html

```diff
@@ -6,6 +6,14 @@
     </head>
     <body>
         <h1>Naija Market Place: Your online one-stop shop</h1>
+            <nav>
+                <div>
```

Comment:
The div within the nav tag is unnecessary. The ul tag within the nav tag should be just fine!